packages/framework/src/Resources/styles/admin/components/wrap/divider.less

```diff
@@ -0,0 +1,17 @@
+@wrap-border-gap: @item-gap;
+@wrap-border-inner-gap-small: 15px;
+
+.wrap-divider {
+    padding-bottom: @wrap-border-gap;
+    margin-bottom: @wrap-border-gap;
+
+    border-bottom: 1px solid @color-border-dark;
+
+    &:last-child {
```

Comment:
this could be changed into `last-of-type` so only div elements would be considered. 
example:
url: /admin/category/edit/2
in the form is last element hidden input tag and the last visible element shows the bottom border
![image](https://user-images.githubusercontent.com/39240194/48778898-adedcf00-ecd6-11e8-8bd7-fb76c8b9b7de.png)